.github/workflows/make_bundle_conda.yml

```diff
@@ -0,0 +1,361 @@
+on:
+  push:
+    # Sequence of patterns matched against refs/tags
+    tags:
+      - "v*" # Push events to matching v*, i.e. v1.0, v20.15.10
```

Comment:
This will catch  rc1 and all that right? Just double checking.

Pinging @sofroniewn

Reply:
yes it will - they always have format like `v0.4.13rc3`